### ISSUE TITLE
Incorrect CSS syntax highlighting

### ISSUE THREAD

**Issue by xfq** [tabatkins/bikeshed#1134]
Comparing https://www.w3.org/TR/2017/CR-css-grid-1-20170509/#example-3828fb4f and http://pygments.org/demo/6660103/, the first one is not highlighted correctly.

**Comment by tabatkins** [comment]
Yeah, I know I've got something wrong in my CSS lexer.  (I had to write a custom one, because the pygments one is based on knowing all the property names/etc beforehand, and can't handle arbitrary CSS; it thus fails pretty badly at highlighting our specs.)
